packages/vm/src/vm.ts

```diff
@@ -59,7 +59,9 @@ export class VM {
    * set to public due to implementation internals
    * @hidden
    */
-  public readonly _emit: (topic: string, data: any) => Promise<void>
+  public readonly _emit: (topic: keyof VMEvents, data: any): Promise<void> {
```

Comment:
```suggestion
  public readonly _emit: (topic: keyof VMEvents, data: unknown): Promise<void> {
```
THis is slightly more typesafe and best practice to be unknown but 🤷🏽